suite-common/wallet-core/src/fiat-rates/fiatRatesReducer.ts

```diff
@@ -51,7 +52,8 @@ export const prepareFiatRatesReducer = createReducerWithExtraDeps(
 
                 state[rateType][fiatRateKey] = {
                     ...currentRate,
-                    rate: action.payload,
+                    rate: action.payload.rate,
+                    lastTickerTimestamp: action.payload.lastTickerTimestamp,
```

Comment:
The `lastTickerTimestamp` has to be multiplied by 1000.

![Screenshot 2024-03-11 at 10 06 24](https://github.com/trezor/trezor-suite/assets/33235762/8118b1c8-647c-4d2b-aca2-1cf54b9705dd)
![Screenshot 2024-03-11 at 10 05 17](https://github.com/trezor/trezor-suite/assets/33235762/cff6ddfb-4080-4f0e-bd08-abc88f3f3e74)
![Screenshot 2024-03-11 at 10 05 11](https://github.com/trezor/trezor-suite/assets/33235762/a5466762-f9fd-4c56-9838-c01be318e678)

Reply:
6d48adb9437ade0dafe5b6ae047d73050e9fdded